### ISSUE TITLE
How to store key:subkey in kubernetes

### ISSUE THREAD

**Issue by VenkateshSrini** [SteeltoeOSS/Steeltoe#631]
I have a configuration settings as below 
```
"DbConnection": {
    "connectionstring": "server=localHost;user=abc;password=pwd123"
  }
```
I'm using the Steeltoe Kubernetes to read the secrets as I like that approach of reading secrets very much. My current code in startup.cs is 
`services.Configure<DbOptions>(Configuration.GetSection("DbConnection")`
How ever I'm unable to set the secrets in Kubernetes as 
`DbConnection:connectionstring : server=localHost;user=abc;password=pwd123
`
I tried setting
`DbConnection__connectionstring : server=localHost;user=abc;password=pwd123`
then the IConfiguration is coming as DbConnection__connectionstring. So please let me know how to set the same

**Comment by TimHess** [comment]
Thanks for raising this issue, it was an oversight in the initial implementation, but should be addressed by #584 and is expected to ship with 3.1.0. It can be tested with a recent build from the [dev feed](https://pkgs.dev.azure.com/dotnet/Steeltoe/_packaging/dev/nuget/v3/index.json)

**Comment by VenkateshSrini** [comment]
What is the expected time line for 3.1.0? My organization does not allow us to use dev feed. It is blocked. So If I take the master branch and the source code as reference will this issue be resolved

**Comment by TimHess** [comment]
We're expecting to have a release candidate within the next couple weeks. Building from `main` would also work

**Comment by jkonicki** [comment]
Addressed in #584.  Closing
